src/ui/common/src/components/tables/virtualizedTable.tsx

```diff
@@ -0,0 +1,252 @@
+import { Box, Typography } from '@mui/material';
+import { styled, Theme } from '@mui/material/styles';
+import TableCell from '@mui/material/TableCell';
+import clsx from 'clsx';
+import * as React from 'react';
+import {
+  AutoSizer,
+  Column,
+  Table,
+  TableCellRenderer,
+  TableHeaderProps,
+} from 'react-virtualized';
+
+const classes = {
+  flexContainer: 'ReactVirtualizedDemo-flexContainer',
+  tableRow: 'ReactVirtualizedDemo-tableRow',
+  tableRowHover: 'ReactVirtualizedDemo-tableRowHover',
+  tableCell: 'ReactVirtualizedDemo-tableCell',
+  noClick: 'ReactVirtualizedDemo-noClick',
+};
+
+const columnWidthMultiplier = 13;
+
+const styles = ({ theme }: { theme: Theme }) =>
+  ({
+    // temporary right-to-left patch, waiting for
+    // https://github.com/bvaughn/react-virtualized/issues/454
+    '& .ReactVirtualized__Table__headerRow': {
```

Comment:
Should we at least make these class names `const`s for the time being as well?

Reply:
styles is a const, so I think it's fine to leave them as is.

My suggestion here would be to move the styles into a file of their own.
so we'd have virtualizedTable.tsx and virtualizedTable.styles.tsx.

Then the styles can be exported and shared amongst other components. I'd also be sure to export the styles from common (src/index.ts) as well. If we haven't done so already, the VirtualTable component should be in there as well.